examples/lines_bars_and_markers/linestyles.py

```diff
@@ -3,31 +3,44 @@
 Linestyles
 ==========
 
-This examples showcases different linestyles copying those of Tikz/PGF.
+This example showcases different linestyles copying those of Tikz/PGF.
+Linestyle can be provided as simple as *solid* , *dotted* or *dashed*.
```

Comment:
Unclear whether this means I can use the word `solid` as an argument to Linestyle or this is talking about lines.